docs/product-installation-guide/src/main/asciidoc/installer-run-cli-proc.adoc

```diff
@@ -1,306 +1,79 @@
 
 .Procedure
 . In a terminal window, navigate to the directory where you downloaded the installer file and enter the following command:
 +
 [source]
 ----
 java -jar rhdm-installer-7.0.0.CR1.jar -console
+
 ----
 +
-The command-line interactive process will start and display the End-User license agreement. 
-. Read the licensem the enter 1 to continue:
-. 
+The command-line interactive process will start and display the End-User License Agreement. 
 +
 [source]
 ----
 press 1 to continue, 2 to quit, 3 to redisplay.
 ----
-
-. Enter `1` to begin the installation and type in the parent directory of an existing {EAP} installation.
+. Read the license agreement then enter `1` to continue:
 +
 [source]
 ----
 The location below must specify the JBOSS_HOME of an existing {EAP} installation.
 
-[/home/user/BPMSuite-VERSION/jboss-eap-6.4]
-----
-+
-The installer will verify the location of the {EAP} installation at the provided location. Enter `1` to confirm and continue.
-
-. _Optional_: create a user for the management console of {EAP} (ManagementRealm):
-+
-[source]
-----
-Create an administrative user
-This user will be added to the host container's management realm for administrative purposes. It can be used to access the management console, the management CLI or other applications secured in this realm.
-----
-+
-Enter `0` to skip creating a new administrative user or `1` to create one. If you do decide to create one, then follow these steps:
-+
-.. Enter a user name:
-+
-[source]
-----
-Admin username: [admin]
-----
-.. Create and confirm a password for the user of the {EAP} management console:
-+
-[source]
-----
-The password must have at least 8 characters, and contain at least one number and one non-alphanumeric symbol (not including the character &).
-
-Admin password: []
-
-Confirm admin password: [******************************]
+[/home/user/RHDM-7.0.0/jboss-eap-7.1]
 ----
-
 +
-After this user has been created successfully, continue to the next step.
-
-. Create a {PRODUCT} administrator user.
+. Enter the parent directory of an existing {EAP} 7.1 installation.
 +
-[source]
-----
-Create a Business Process Management Suite Admin User
-Create a BPM Suite admin user. The user will be added to the ApplicationRealm, and can be used to access the Business Central Console. The User will be assigned the 'admin' application roles. The BPM Suite username cannot be any of the following: 'admin', 'analyst', 'user', 'manager' or 'developer'.
-
-BPM Suite username: [bpmsAdmin]
-----
+The installer will verify the location of the {EAP} installation at the location provided. Enter `1` to confirm and continue.
+. Select the components that you want to install by entering the numbers next to the components, then enter `0` to confirm your selection.
+. Press `1` to continue.
+COMMENT: Need information on the TomCat8 options.
```

Comment:
They setup kie-server for tomcat and controller for kie-server
I think they are self-explanatory